install/install.md

```diff
@@ -215,8 +279,10 @@ Windows users can now tap into GPU accelerated data science on their local machi
 	```
 
 <br/>
+<div id="wsl2-docker"></div>
+
+### **WSL2 Docker Desktop Install**
 
-### **WSL Docker Desktop**
 1. Install WSL2 and the Ubuntu 22.04 package [using Microsoft's instructions](https://docs.microsoft.com/en-us/windows/wsl/install){: target="_blank"}.
 2. Install the [latest NVIDIA Drivers](https://www.nvidia.com/download/index.aspx){: target="_blank"} on the Windows host.
 3. Install latest Docker Desktop for Windows 
```

Comment:
In the steps below, has anyone tried using WSL in Ubuntu 22.04.2 without docker desktop installed? Just want to confirm it works - have never tried personally.

Reply:
I just tried in my Ubuntu 22.04 WSL2 instance that was installed through the MS Store and docker doesn't seem to be included:

```
sudo service docker start
>>> docker: unrecognized service

systemctl enable docker
>>> Failed to enable unit, unit docker.service does not exist
```

I'm leaning hard that we should just strip this part from the docs and recommend installing Docker Desktop regardless.

Reply:
Apologies for any confusion! I'm talking about [this line specifically](https://github.com/rapidsai/docs/pull/377/files/689e933c2bd3afee3783efb7095e0b240450e01c#diff-b1b679a811f38c824cce215c29f17a75472d763031023c125d190e579f723b34R290). I blame GitHub for not letting me comment on unchanged lines 😅 

>    - If running WSL2's Ubuntu 22.04.2 LTS containers, it already has Docker installed.  Log into your WSL2 machine and run `sudo service docker start`, skip Step # 4 and go to Step # 5.

Reply:
My experience has been that the above line is not true, I tried it in the previous comment, with no success.

Reply:
thanks that cleared it up perfectly.